.github/workflows/pr-realm-js.yml

```diff
@@ -239,6 +237,18 @@ jobs:
         if: ${{ runner.os == 'Windows' }}
         uses: ilammy/msvc-dev-cmd@v1
 
+      - name: ccache
+        uses: hendrikmuhs/ccache-action@v1
+        #TODO: figure out using ccache or equivalent on windows
+        if: ${{ matrix.variant.environment == 'react-native' }}
+        with:
+          key: ${{ runner.os }}-${{ matrix.variant.os }}-test
```

Comment:
Is it on purpose that we don't reuse the cache from the job above? If so, I'd like a comment on why we can't / won't make sense to reuse the key from job above.

Reply:
No reason other than simplicity. The build job ccache cache key uses matrix.variant.arch which doesn't exist in this matrix. It could make sense to introduce a matching variable in this matrix, but I don't know what to set it to for other reach-native targets. If you have an idea what those could be then I will add them and share the cache across the jobs.

Reply:
Can you simply set it to `undefined`, `null` or equivalent?